src/Mod/Path/PathScripts/PathSimulatorGui.py

```diff
@@ -382,6 +383,9 @@ def CreateToolProfile(self, tool, dir, pos, rad):
         yp = pos[1]
         zp = pos[2]
         h = tool.CuttingEdgeHeight
+        if h <= 0.0:#set default if user fails to avoit Freeze
```

Comment:
Should be **avoid**

Reply:
yes, you are right. shall I fix it and request again?

Reply:
If changing that line, please also make PEP8 happy:  
if h <= 0.0:  # set default if user fails, to avoid freeze